src/Utilities/Microsoft.Build.Utilities.csproj

```diff
@@ -31,6 +32,10 @@
     <PackageReference Include="System.Text.Encoding.CodePages" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <PackageReference Include="Microsoft.IO.Redist" />
```

Comment:
These references have setup impact, right? Can you add them to the VSIX?

Reply:
Good point.